src/modules/file/file.repository.ts

```diff
@@ -129,8 +133,8 @@ export class SequelizeFileRepository implements FileRepository {
     private fileModel: typeof FileModel,
   ) {}
 
-  async findAll(): Promise<Array<File> | []> {
-    const files = await this.fileModel.findAll();
+  async findAll(query = {}): Promise<Array<File> | []> {
```

Comment:
Do not do this, do concrete methods where the conditions are specified and name it properly. 
This is the same like bypassing the repository layer.